tests/core/numerics/interpolator/interpolator_test.py

```diff
@@ -43,5 +43,15 @@ def main():
 
 
 
+def interpolate_datasets(beforeTime, afterTime, interp_time, beforeData, afterData):
+    alpha = (interp_time - beforeTime) / (afterTime - beforeTime)
+    assert beforeData.shape == afterData.shape
+    interpData = afterData.copy()
+    assert interpData.ndim == 1 # update for > 1d interpolation values
+    if interpData.ndim == 1:
```

Comment:
the formula is valid for any dimension so this assert is superfluous

Reply:
and the "if" too